code/__HELPERS/mobs.dm

```diff
@@ -127,11 +127,12 @@ Proc for attack log creation, because really why not
 
 /proc/add_logs(mob/user, mob/target, what_done, object=null, addition=null)
 	var/newhealthtxt = ""
-	var/turf/attack_location = get_turf(target)
-	var/coordinates = "([attack_location.x],[attack_location.y],[attack_location.z])"
+	var/coordinates = ""
 	if (target && isliving(target))

```

Comment:
The only time a ghost should ever be getting hit is when admins (or a wizard) make them visible, and even then it doesn't matter at all because you can't do anything to ghosts.